apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -72,7 +73,18 @@ function SectionTable(props: SectionTableProps) {
 
     const [activeColumns] = useColumnStore((store) => [store.activeColumns]);
     const [activeTab] = useTabStore((store) => [store.activeTab]);
+    const { isMilitaryTime } = useTimeFormatStore()
     const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT})`);
+    const [isCompact, setIsCompact] = useState(false);
+    const buttonRowRef = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        const observer = new ResizeObserver(([entry]) =>
+          setIsCompact(entry.contentRect.width < 750)
+        );
+        observer.observe(buttonRowRef.current!);
```

Comment:
Do you not see that there's a warning here of `Forbidden non-null assertion`? If not, we need to figure out how to fix that.

You can just check if `buttonRowRef` is `null` and return (+ `console.error`) if it is.